File: TimeSpent.Web/Binding/Models/TimeEntryModel.js
(function(ts){
    var TimeEntryModel = function () {

        var self = this;
        self.TimeEntryId ='';
        self.Duration ='';
        self.Date = '';
        self.User = '';
        self.ProjectId = '';
        self.CategoryId = '';
        self.Description = '';
        
    }
    ts.TimeEntryModel = TimeEntryModel;
}(window.TimeSpent));

File: TimeSpent.Web/Models/TimeEntryModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TimeSpent.Web.Models
{
    public class TimeEntryModel
    {
        public int TimeEntryId { get; set; }
        public float Duration { get; set; }
        public DateTime Date { get; set; }
        public int User { get; set; }
        public int Project { get; set; }
        public int Category { get; set; }
        public string Description { get; set; }
    }
}